q2_metadata/plugin_setup.py

```diff
@@ -113,15 +113,21 @@ def _2(ff: ArtificialGroupingFormat) -> (qiime2.Metadata):
     inputs={},
     parameters={'metadata': MetadataColumn[Categorical],
                 'n_columns': Int,
-                'column_name_prefix': Str,
-                'column_value_prefix': Str},
+                'md_column_name_prefix': Str,
+                'md_column_values_prefix': Str,
+                'encode_sample_size': Bool
+                },
     parameter_descriptions={
         'metadata': ('Categorical metadata column to shuffle.'),
         'n_columns': 'The number of shuffled metadata columns to create.',
-        'column_name_prefix': ('Prefix to use in naming the shuffled '
-                               'metadata columns.'),
-        'column_value_prefix': ('Prefix to use in naming the values in the '
-                                'shuffled metadata columns.')},
+        'md_column_name_prefix': ('Prefix to use in naming the shuffled '
+                                  'metadata columns.'),
+        'md_column_values_prefix': ('Prefix to use in naming the values in the'
```

Comment:
```suggestion
        'md_column_values_prefix': ('Prefix to use in naming the values in the '
```

Reply:
You need an extra space between each new line in these parameter descriptions, otherwise when it's rendered on the command line, the words get mashed together like this:
<img width="566" alt="Screen Shot 2023-07-25 at 4 50 22 PM" src="https://github.com/qiime2/q2-metadata/assets/54517601/639a45e9-c75a-4187-9cdb-6ea0a38c17e3">